repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,113 @@
+=== <<body-extractor-to-header.adoc#,Body Extractor to Header Filter>>
+*IN PROGRESS*
 
-<<rackspace-auth-user.adoc#,Rackspace Auth User Filter>>::
-    For identity usage for rate limiting and user access events.
+=== <<body-patcher.adoc#,Body Patcher Filter>>
+*IN PROGRESS*
 
-<<scripting.adoc#,Scripting Filter>>::
-    Enables users to write custom filters for Repose using a
-    variety of scripting languages.
+=== <<compression.adoc#,Compression Filter>>
+*IN PROGRESS*
+
+=== <<content-type-stripper.adoc#,Content Type Stripper Filter>>
+*IN PROGRESS*
+
+=== <<cors.adoc#,CORS Filter>>
+Enables users to add support for Cross-Origin Resource Sharing requests.
+
+=== <<derp.adoc#,Delegation Response Processor (DeRP) Filter>>
+*IN PROGRESS*
+
+=== <<destination-router.adoc#,Destination Router Filter>>
+*IN PROGRESS*
+
+=== <<forwarded-proto.adoc#,Forwarded Protocol Filter>>
+*IN PROGRESS*
+
+=== <<header-normalization.adoc#,Header Normalization filter>>
+*IN PROGRESS*
+
+=== <<header-translation.adoc#,Header Translation Filter>>
+*IN PROGRESS*
+
+=== <<header-user.adoc#,Header User Filter>>
+*IN PROGRESS*
+
+=== <<herp.adoc#,Highly Efficient Record Processor (HERP) Filter>>
+*IN PROGRESS*
+
+=== <<ip-user.adoc#,IP User Filter>>
+*IN PROGRESS*
+
+=== <<iri-validator.adoc#,IRI Validator Filter>>
+*IN PROGRESS*
+
+=== <<keystone-v2.adoc#,Keystone V2 Filter>>
+*IN PROGRESS*
+
+=== <<keystone-v2-basic-auth.adoc#,Keystone v2 Basic Auth Filter>>
+*IN PROGRESS*
+
+=== <<merge-header.adoc#,Merge Header Filter>>
+*IN PROGRESS*
+
+=== <<openstack-identity-v3.adoc#,Openstack Identity v3 Filter>>
+*IN PROGRESS*
+
+=== <<rackspace-auth-user.adoc#,Rackspace Auth User Filter>>
+For identity usage for rate limiting and user access events.
+
+=== <<rate-limiting.adoc#,Rate Limiting Filter>>
+*IN PROGRESS*
+
+=== <<slf4j-http-logging.adoc#,SFL4J HTTP Logging Filter>>
+*IN PROGRESS*
+
+=== <<translation.adoc#,Translation Filter>>
+*IN PROGRESS*
+
+=== <<uri-normalization.adoc#,URI Normalization Filter>>
+*IN PROGRESS*
+
+=== <<uri-stripper.adoc#,URI Stripper Filter>>
+*IN PROGRESS*
+
+=== <<uri-user.adoc#,URI User Filter>>
+*IN PROGRESS*
+
+=== <<url-extractor-to-header.adoc#,URL Extractor to Header Filter>>
+*IN PROGRESS*
+
+=== <<valkyrie-authorization.adoc#,Valkyrie Authorization Filter>>
+*IN PROGRESS*
+
+=== <<versioning.adoc#,Versioning Filter>>
+*IN PROGRESS*
+
+== Repose Extensions Filter Bundle
+=== <<api-validator.adoc#,API Validator Filter>>
+*IN PROGRESS*
+
+=== <<simple-rbac.adoc#,Simple Role Based Access Control (RBAC) Filter>>
+*IN PROGRESS*
+
+== Repose Experimental Filter Bundle
+=== <<echo.adoc#,Echo Filter>>
+*IN PROGRESS*
+
+=== <<exception.adoc#,Exception Filter>>
+*IN PROGRESS*
+
+=== <<scripting.adoc#,Scripting Filter>>
+Enables users to write custom filters for Repose using a variety of scripting languages.
+
+=== <<servlet-contract.adoc#,Servlet Contract Filter>>
+*IN PROGRESS*
+
+=== <<tightly-coupled.adoc#,Tightly Coupled Filter>>
```

Comment:
Drop these filters and don't split this page by filter bundle.

Eventually, we will break these testing filters off into their own bundle.